libmachine/host.go

```diff
@@ -167,7 +155,7 @@ func (h *Host) Start() error {
 		return err
 	}
 
-	return utils.WaitFor(h.MachineInState(state.Running))
+	return utils.WaitFor(drivers.MachineInState(h.Driver, state.Running))
```

Comment:
Yeah I agree it's hairy.  Maybe we could have an additional short-hand method available in `utils`, like `utils.WaitForDriverState(h.Driver, state.Running)`.  I just don't want to re-implement "run an action, then wait for the state that action is trying to reach to be fulfilled" in different places.